CMakeLists.txt

```diff
@@ -20,7 +20,28 @@ else()
     find_package(Threads)
 
     set(CMAKE_CXX_FLAGS
-      "${CMAKE_CXX_FLAGS} -g -std=c++11 -Wall -Wpedantic")
+      "${CMAKE_CXX_FLAGS} -std=c++11 -Wall -Wpedantic")
+endif()
+
+if (${VARIANT} STREQUAL "coverage")
```

Comment:
Not sure how this popped up after the code was merged but its being looked into, thanks.